resources/environments/resource_azure_image_terms.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2023 Cloudera. All Rights Reserved.
+//
+// This file is licensed under the Apache License Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, either express or implied. Refer to the License for the specific
+// permissions and limitations governing your use of the file.
+
+package environments
+
+import (
+	"context"
+
+	"github.com/cloudera/terraform-provider-cdp/cdp-sdk-go/cdp"
+	"github.com/cloudera/terraform-provider-cdp/cdp-sdk-go/gen/environments/client/operations"
+	environmentsmodels "github.com/cloudera/terraform-provider-cdp/cdp-sdk-go/gen/environments/models"
+	environments "github.com/cloudera/terraform-provider-cdp/resources"
+	"github.com/cloudera/terraform-provider-cdp/utils"
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+var (
+	_ resource.Resource = &azureImageTermsResource{}
+)
+
+type azureImageTermsResource struct {
+	client *cdp.Client
+}
+
+type azureImageTermsResourceModel struct {
+	ID       types.String `tfsdk:"id"`
+	Accepted types.Bool   `tfsdk:"accepted"`
+}
```

Comment:
even if it's just two fields, for the sake of unity, could you please extract the model, like we did for the rest of the resources? 🙂